crypto/hd/algorithm_test.go

```diff
@@ -0,0 +1,132 @@
+package hd
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	hdwallet "github.com/miguelmota/go-ethereum-hdwallet"
+
+	amino "github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+
+	ethermint "github.com/evmos/ethermint/types"
+	cryptocodec "github.com/evmos/evmos/v11/crypto/codec"
+	enccodec "github.com/evmos/evmos/v11/encoding/codec"
+)
+
+var TestCodec amino.Codec
+
+func init() {
+	cdc := amino.NewLegacyAmino()
+	cryptocodec.RegisterCrypto(cdc)
+
+	interfaceRegistry := types.NewInterfaceRegistry()
+	TestCodec = amino.NewProtoCodec(interfaceRegistry)
+	enccodec.RegisterInterfaces(interfaceRegistry)
+}
+
+const (
+	mnemonic = "picnic rent average infant boat squirrel federal assault mercy purity very motor fossil wheel verify upset box fresh horse vivid copy predict square regret"
+
+	// hdWalletFixEnv defines whether the standard (correct) bip39
+	// derivation path was used, or if derivation was affected by
+	// https://github.com/btcsuite/btcutil/issues/172
+	hdWalletFixEnv = "GO_ETHEREUM_HDWALLET_FIX_ISSUE_179"
```

Comment:
it says 172 in the comment but 179 in the string?
```suggestion
	// https://github.com/btcsuite/btcutil/issues/179
	hdWalletFixEnv = "GO_ETHEREUM_HDWALLET_FIX_ISSUE_179"
```